src/config/dark-sites.config

```diff
@@ -112,14 +126,17 @@ dplay.com
 dpreview.com
 draculatheme.com
 draftkings.com
+drunkenslug.com
 duelingnexus.com
 ecobee.com/consumerportal
 editor.freepik.com
 editor.method.ac
 egee.io
 eggsy.codes
+elderscrolls.fandom.com
```

Comment:
![](https://i.imgur.com/8BrXShe.png) Sneaky site :eyes:

Reply:
It's White.

Reply:
ah i didn't get that cookie notice, maybe because of ublock.